sets/showspecific.txt

```diff
@@ -594,16 +670,27 @@ Cybodies.
 The Galactic Pretty Boy.
 Kiraboshi.
 # STEINS;GATE
+@Channel.
 CERN.
 Dr. Pepper.
 El. Psy. Congroo.
 Feyris nyan-nyan.
 Gel-bananas.
+Hacking to the Gate.
 Hououin Kyouma.
+Human is Dead. Mismatch.
+A mashup of English AND German.
 Lab coats.
-Maddo Scientistu.
+Maddo Scientisto.
+Phone Microwave.
+Sunuvabitch!
+Shining Finger.
```

Comment:
Shining Finger is from G Gundam. It should go back to Misc.

Reply:
I suppose that's okay, but I feel like more people see it as a Steins;Gate reference than anything else.

Also, I suggested a lot of the changes I made here in that giant google doc (because Piru said it was "easier this way"), so you might want to look out for duplicates when merging (that is, if you do decide to merge).